.lintstagedrc.json

```diff
@@ -1,10 +1,4 @@
 {
-  "*.js": [
-    "prettier --write",
-    "eslint"
-  ],
-  "*.ts": [
-    "prettier --write",
-    "eslint"
-  ]
-}
\ No newline at end of file
+  "*.js": ["prettier --write"],
+  "*.ts": ["prettier --write"]
```

Comment:
Lint stage was failing, needs to be addressed in another PR, removing this for now but happy to add it back and address this here if we think its necessary.